### ISSUE TITLE
Equals that takes the raw type T

### ISSUE THREAD

**Issue by StuporHero** [yegor256/cactoos#1173]
Having to wrap raw objects in `Scalar<T>` is a little cumbersome if I've already got the raw type. Would it be possible to add a constructor to `Equals` that wraps raw inputs with lambdas to save a little trouble?

Edit added by @victornoel: when it is done, let's solve the problem expressed in #1174.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@stuporhero[/z](https://www.0crat.com/u/stuporhero) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by victornoel** [comment]
@StuporHero could you show a code example to clarify what you want to do?

**Comment by StuporHero** [comment]
@victornoel

What must be done now:
```java
ComparableType a = new ComparableType();
ComparableType b = new ComparableType();
Equals e = new Equals(() -> a, () -> b);
```

What I'd like to be able to do:
```java
ComparableType a = new ComparableType();
ComparableType b = new ComparableType();
Equals e = new Equals(a, b);
```

Just add a Constructor like this:
```java
    /**
     * Ctor.
     * @param source The first scalar to compare.
     * @param compared The second scalar to compare.
     */
    public Equals(final T source, final T compared) {
        this(() -> source, () -> compared);
    }
```

**Comment by victornoel** [comment]
@StuporHero ok, it's clear now, I thought you were asking for something more complex ^^ Thanks

Let's do that, and also let's take a look around the other classes in the `scalar` package to see if they have the same problem and let's fix it.

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1173#issuecomment-691663165))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1173](https://github.com/yegor256/cactoos/issues/1173) is [in scope](http://www.zerocracy.com/policy.html#14) for 7s
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231173%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by victornoel** [comment]
@StuporHero let's also solve #1174 at the same time here, I updated the description for it :)

**Comment by victornoel** [comment]
Fixed in #1546

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1173` is not assigned, can't get performer
